rust/arrow/src/json/writer.rs

```diff
@@ -42,7 +45,39 @@
 //! );
 //! ```
 //!
-//! Serialize record batches into line-delimited JSON bytes:
+//! ## Writing JSON formatted byte streams
```

Comment:
The doc examples are probably the best way to see how to use this structure and what the changes looked like